lib/paper_trail/compatibility.rb

```diff
@@ -17,8 +17,8 @@ module PaperTrail
   # newer rails versions. Most PT users should avoid incompatible rails
   # versions.
   module Compatibility
-    ACTIVERECORD_GTE = ">= 4.2"
-    ACTIVERECORD_LT = "< 6.1"
+    ACTIVERECORD_GTE = ">= 5.2" # enforced in gemspec
```

Comment:
Would it make sense to freeze these?

Reply:
Yes, but they're already frozen by the `frozen_string_literal` setting, right?

Reply:
Oh right, thanks I missed that